scan/cli.go

```diff
@@ -230,13 +252,38 @@ func createGenericAuditCmd(c *cli.Context) (*audit.GenericAuditCommand, error) {
 	if c.String("working-dirs") != "" {
 		auditCmd.SetWorkingDirs(splitAndTrim(c.String("working-dirs"), ","))
 	}
+	auditCmd.GraphBasicParams = &xrCmdUtils.GraphBasicParams{}
+	auditCmd.SetServerDetails(serverDetails).
+		SetExcludeTestDependencies(c.Bool(cliutils.ExcludeTestDeps)).
+		SetOutputFormat(format).
+		SetUseWrapper(c.BoolT(cliutils.UseWrapper)).
+		SetInsecureTls(c.Bool(cliutils.InsecureTls)).
+		SetNpmScope(c.String(cliutils.DepType)).
+		SetPipRequirementsFile(c.String(cliutils.RequirementsFile))
+	return auditCmd, err
+}
 
-	return auditCmd.SetExcludeTestDependencies(c.Bool(cliutils.ExcludeTestDeps)).
-			SetUseWrapper(c.BoolT(cliutils.UseWrapper)).
-			SetInsecureTls(c.Bool(cliutils.InsecureTls)).
-			SetNpmScope(c.String(cliutils.DepType)).
-			SetPipRequirementsFile(c.String(cliutils.RequirementsFile)),
-		err
+func createCurtainCmd(c *cli.Context) (*curation.Command, error) {
```

Comment:
This function seems redundant. Let's move its content into the `CurationAuditCmd` function.